src/mon/OSDMonitor.cc

```diff
@@ -7943,7 +7943,7 @@ int OSDMonitor::check_pg_num(int64_t pool,
 
   for (const auto& [pool_id, pool_info] : osdmap.get_pools()) {
     // Check only for pools affected by crush rule
-    if (crush_pool_ids.contains(pool_id)) {
+    if (crush_pool_ids.count(pool_id)) {
```

Comment:
Sorry, I've missed it last time – this commit is basically a fixup. When backporting the typical practice is to squash such changes while informing the main commit about any customization and the reason for it.

Reply:
Ack